mobile/test/common/extensions/filters/http/platform_bridge/BUILD

```diff
@@ -31,6 +31,7 @@ envoy_extension_cc_test(
         "platform_bridge_filter_integration_test.cc",
     ],
     extension_names = ["envoy.filters.http.platform_bridge"],
+    rbe_pool = "6gig",
```

Comment:
this wont work out of the box - what it actually needs to be is:

```starlark
exec_properties = {"Pool": "6gig"}
```

probably it makes sense to set up a macro - so that you can set it as `rbe_pool` - see eg

https://github.com/envoyproxy/envoy/blob/4c0975228ab82e8c4965f351a4cbf771a027617f/bazel/envoy_library.bzl#L117-L120